### PR TITLE
20 fix breaking tests

### LINKED ISSUE #20: as a developer, the hambda `services/process` test suite all passes in circle ci

### PR DESCRIPTION
**Pull request by DanielSeehausen**
second part of 20
closes #20

### CODE REVIEW

File: services/device/test/seeds/devices.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  deviceId: uuidv4(),
+  name: 'minimal device',
+  users: [user.userId],
+  access: [{
+    users: [user.userId],
+    groupName: 'owners',
+    rights: '11111'
+  }],
+  streams: [
+    {streamId: 'streamData1'},
```
Comment (vdiezel) [inline services/device/test/seeds/devices.js]: I mislead you here when we did pairing, the streamId prop of the objects in the `streams`-array should be just `id`. Really sorry here!

File: services/process/test/processes.test.js [hunk 2]
```diff
@@ -1,1489 +1,623 @@
...
-      expect(insertedProcess.createdBy).toBe(userId);
-      expect(insertedProcess.comment).toBe('processComment');
-      expect(insertedProcess.deviceDescription[0].streamId).toBe('streamData1');
-      expect(insertedProcess.deviceDescription[1].streamId).toBe('streamData2');
+      expect(body.name).toBe(process.name)
+      expect(body.deviceId).toBe(device.deviceId)
+      expect(body.cloneHistory.length).toBe(0)
+      expect(body.createdBy).toBe(user.userId)
+      expect(body.comment).toBe(process.comment)
+      expect(body.deviceDescription[0].streamId).toBe(device.streams[0].streamId)
```
Comment (vdiezel) [inline services/process/test/processes.test.js]: streamId --> id

File: services/process/test/seeds/processes.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+  CREATORASVIEWER: (processData) => {
+    const groups = processData.access
+    const operators = groups.find(group => group.name === 'viewers')
+    operators.users.push(processData.users[0])
+    return { access: groups }
+  }
+}
+
+function applyTraits(originalData, traits) {
```
Comment (vdiezel) [inline services/process/test/seeds/processes.js]: love this!

File: services/device/test/seeds/devices.js (hunk 1 above)
Comment (DanielSeehausen) [reply services/device/test/seeds/devices.js]: thank you - fixed

File: services/process/test/processes.test.js (hunk 2 above)
Comment (DanielSeehausen) [reply services/process/test/processes.test.js]: resolved thank you

**Pull request merged**